app/screens/navigation.ts

```diff
@@ -678,10 +678,17 @@ export async function bottomSheetWithTeamList({title, teams, dimensions, renderC
     const NO_TEAMS_HEIGHT = 392;
     const maxHeight = Math.round((dimensions.height * 0.9));
 
+    const ADDITIONAL_BOTTOM_MARGIN = 34;
+    const BOTTOM_SHEET_TOP_PADDING = 20; // bottomSheet paddingTop
```

Comment:
For sure, I would move the BOTTOM_SHEET_TOP_PADDING to the bottom sheet component, and make the style use the constant. The other margin, I am not sure where it comes from, so not sure if this is the right place to define it.